packages/components/src/text/text.tsx

```diff
@@ -12,10 +12,12 @@ type Weight = NonNullable<Variants['weight']>
 
 type Props = {
   children: React.ReactNode
+  size?: 32 | 27 | 19 | 15 | 13 | 11 | undefined
   color?: ColorTokens
   truncate?: boolean
   wrap?: false
 } & (
+  | { size: 32; weight?: Weight }
```

Comment:
this variant does not exist

Reply:
I checked the designs and the largest (80) Avatar is using `27`. Also, Text should be in sync with Typography in Figma.

Reply:
I did that for emoji, by approximation, will use `27` instead.

<img width="530" alt="image" src="https://user-images.githubusercontent.com/13265126/232156594-b241ab55-8161-423b-90f7-322a3d7e4f14.png">

Reply:
Got it. The consumption of emojis is still unknown to me at this point. I feel Twemoji can behave differently from native ones if we go down that road.

Reply:
removed at https://github.com/status-im/status-web/pull/371/commits/0e53bbceeacebbd2c8414b3404e2573e2d9fcb4e